examples/platform_services/ios/Runner/AppDelegate.m

```diff
@@ -6,11 +6,34 @@
 
 #import <Flutter/Flutter.h>
 
-@implementation AppDelegate {
-}
-
+@implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    return YES;
+  FlutterDartProject* project = [[FlutterDartProject alloc] initFromDefaultSourceForConfiguration];
+  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+  FlutterViewController* controller = [[FlutterViewController alloc] initWithProject:project
```

Comment:
Our samples/templates usually instantiate the FlutterViewController in Main.storyboard and don't have this self.window assignment line. Then they say "FlutterViewController* controller = (FlutterViewController*)self.window.rootViewController;" This is fewer lines of code, though it does have a cast. Is there a reason why you're doing it differently here?